### PR TITLE
version 0.1.4

### PR DESCRIPTION
**Pull request by BrentDorsey**
- updated recipe to version 0.1.4.
- expanded description
- implemented pin_compatible to resolve overdepending

Thanks,

[BrentDorsey](https://github.com/BrentDorsey)
[hill-country-machine-learning](https://github.com/hill-country-machine-learning)

### CODE REVIEW

**Comment by BrentDorsey** [comment]
@pahaz @basnijholt @fernandezcuesta @mariusvniekerk - anyone available to review this PR?

**Comment by basnijholt** [comment]
Looks good but I think that the description is a bit too much. This isn't supposed to be a usage reference but just for installation.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,32 +1,29 @@
...
 
 build:
   noarch: python
   entry_points:
     - sshtunnel = sshtunnel:_cli_main
-  number: 1
-  script: python -m pip install --no-deps --ignore-installed .
-
+  number: 0
+  script: python -m pip install . --no-deps --ignore-installed --verbose
```
Comment (basnijholt) [inline recipe/meta.yaml:17]: Why did you add verbose?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,32 +1,29 @@
...
-    - pip
     - python
-    - setuptools
+    - pip >=18.1
+    - setuptools >=34.0.0
+    - paramiko >=1.15.2
   run:
-    - python
-    - paramiko
+    - {{ pin_compatible('paramiko') }}
```
Comment (basnijholt) [inline recipe/meta.yaml]: Why is this needed? And not just paramiko?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -35,12 +32,228 @@ test:
 about:
   home: http://github.com/pahaz/sshtunnel
   license: MIT
-  license_file: LICENSE
```
Comment (basnijholt) [inline recipe/meta.yaml:38]: Why did you remove this?

**Review by basnijholt — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (BrentDorsey) [reply recipe/meta.yaml]: I implemented pin_compatible to resolve overdepending.  The conda build logs indicate in future versions overdepending will be changed from a warning to an error.

[Pinning at the recipe level](https://github.com/conda/conda-build/blob/master/docs/source/resources/variants.rst#pinning-at-the-recipe-level) has a good description and examples including using lower_bound and upper_bound vs min_pin and max_pin.

File: recipe/meta.yaml (hunk 3 above)
Comment (BrentDorsey) [reply recipe/meta.yaml:38]: The pypi package has a license included in the source archive.  The only time you need to set license_file is when you're including a license in the recipe because the source archive doesn't have a license and license_file should be set to the path of the license file relative to the recipe directory.  This conda-forge feedstock repo doesn't contain a license file.

**Comment by BrentDorsey** [comment]
@basnijholt - This PR is ready to merge whenever you are.

File: recipe/meta.yaml (hunk 2 above)
Comment (basnijholt) [reply recipe/meta.yaml]: I've only ever seen this being used for packages with non-Python code (C for example), because the compiler used for those packages should be the same for the packages downstream (if they need to be compiled too.)

AFAIK paramiko is pure Python.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -35,12 +32,25 @@ test:
...
+  license_family: MIT
   summary: 'Pure Python SSH tunnels'
+  description: |
+    sshtunnel Pure python SSH tunnels
+    -------------
+          Author - [Pahaz Blinov](https://github.com/pahaz)
```
Comment (basnijholt) [inline recipe/meta.yaml]: The Markdown added here isn't parsed somehow in the [README.md](https://github.com/conda-forge/sshtunnel-feedstock/blob/b61f4772d1dcb654d15b0857b738adbf98b90c62/README.md).

**Review by basnijholt — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (BrentDorsey) [reply recipe/meta.yaml]: pin_compatible removed.

File: recipe/meta.yaml (hunk 4 above)
Comment (BrentDorsey) [reply recipe/meta.yaml]: I removed description from the about section but the Current build status section is still not rendering correctly.

Thoughts?

Comment (basnijholt) [reply recipe/meta.yaml]: This is probably a `conda-smithy` bug.

**Comment by basnijholt** [comment]
@BrentDorsey thanks a lot!

**Review by basnijholt — APPROVED** [review]
